IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_MarinePartChamberCommon/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ChamberCommon" UniqueId="63a907b1-a065-46f5-adbe-1bac26deb245" ApplicableType="IfcFacilityPart/IfcMarinePartTypeEnum(.CHAMBER.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarinePartChamberCommon" UniqueId="63a907b1-a065-46f5-adbe-1bac26deb245" ApplicableType="IfcFacilityPart/IfcMarinePartTypeEnum(.CHAMBER.)" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
This ApplicableType-string would probably change to "IfcMarineFacilityPart/CHAMBER" after the redesign of facility part structure.